### ISSUE TITLE
Use Cldr.Territory.country_codes for generating the default territory list for Cldr.HTML.Territory.select

### ISSUE THREAD

**Issue by walu-lila** [elixir-cldr/cldr_html#10]
Currently if the territories option is not set, the default territories list is used. This list is generated by using `Cldr.known_territories()`, however the values returned are not all valid alpha2 country codes. This causes an exception to be raised. For example:
```elixir
(FunctionClauseError) no function clause matching in MyApp.Cldr.Territory.from_territory_code/3
(myapp 0.1.0) lib/cldr/backend.ex:1: Dex.Cldr.Territory.from_territory_code(:AA, "en-GB", :standard)
(cldr_html 1.0.0) lib/cldr_html_territories.ex:241: Cldr.HTML.Territory.name_from_territory/3
(cldr_html 1.0.0) lib/cldr_html_territories.ex:234: Cldr.HTML.Territory.territory_info/2
(elixir 1.11.3) lib/enum.ex:1411: Enum."-map/2-lists^map/1-0-"/2
(elixir 1.11.3) lib/enum.ex:1411: Enum."-map/2-lists^map/1-0-"/2
(cldr_html 1.0.0) lib/cldr_html_territories.ex:218: Cldr.HTML.Territory.territory_options/1
(cldr_html 1.0.0) lib/cldr_html_territories.ex:127: Cldr.HTML.Territory.select/4
```

If  `Cldr.Territory.country_codes()` is used the exceptions are no longer thrown.

**Comment by kipcole9** [comment]
Thanks for the report and apologies for the inconvenience.  The issue arises because `:AA` is actually a valid but reserved and definitely should be presented as an option.  I will update `ex_cldr` to do a better job of excluding reserved (and some other categories) from `Cldr.known_territories/0` but definitely better to use `Cldr.Territory.country_codes()` and will update now.

**Comment by kipcole9** [comment]
As of this [commit](https://github.com/elixir-cldr/cldr_html/commit/b06ca2611394981e0e70fa3649214d9189ff9c99) the default territory list is now `Cldr.Territory.known_country_codes/0`. I have published [ex_cldr_html version 1.0.1](https://hex.pm/packages/ex_cldr_html/1.0.1) with the following changelog entry. Thanks very much for the issue and collaboration.

### Bug Fixes

* Use `Cldr.Territory.country_codes/0` as the default territory list. Previously `Cldr.known_territories/0` was used. However that function returns a list of valid codes, not "known to CLDR" codes (which also needs fixing). Thanks to @walu-lila for the report. Fixes #10.
